src/org/zaproxy/zap/extension/formHandler/ZapAddOn.xml

```diff
@@ -0,0 +1,20 @@
+<zapaddon>
+	<name>Form Handler</name>
+	<version>1</version>
+	<status>alpha</status>
+	<description>This is a test of the form handling information box --ALPHA--</description>
+	<author>ZAP Dev Team</author>
+	<url></url>
+	<changes>First version</changes>
+	<extensions>
+		<extension>org.zaproxy.zap.extension.formHandler.ExtensionFormHandler</extension>
+	</extensions>
+	<ascanrules/>
+	<pscanrules/>
+	<filters/>
+	<files>
+		<file></file>
```

Comment:
This should be removed (core expects a file if the element is present, if empty it warns: Ignoring empty "file" entry in add-on "Form Handler").